src/inference/hmc.jl

```diff
@@ -538,7 +549,12 @@ function HMCState(
 
     # Get the initial log pdf and gradient functions.
     ∂logπ∂θ = gen_∂logπ∂θ(vi, spl, model)
-    logπ = Turing.LogDensityFunction(vi, model, spl, DynamicPPL.DefaultContext())
+    logπ = Turing.LogDensityFunction(
```

Comment:
Seems this is not tested (codecov complains)?

Reply:
Tests have been added :+1: